run-application.sh

```diff
@@ -2,19 +2,7 @@
 
 db_name=school
 
-createdb "$db_name"
-
-psql $db_name -U postgres -c "CREATE TABLE IF NOT EXISTS current_keys(key VARCHAR PRIMARY KEY, serial_number INTEGER UNIQUE)"
-psql $db_name -U postgres -c "CREATE TABLE IF NOT EXISTS total_records(count INTEGER)"
-
-psql $db_name -U postgres -c "insert into total_records (count) select 0 where not exists (select count from total_records)"
-
-psql $db_name -U postgres -c "CREATE TABLE IF NOT EXISTS ordered_entry_index(serial_number INTEGER PRIMARY KEY, entry JSONB)"
-psql $db_name -U postgres -c "ALTER TABLE ordered_entry_index drop column leaf_input"
```

Comment:
Maybe this should be ALTER TABLE ordered_entry_index drop column is exists leaf_input. Would avoid the below warning.

"NOTICE:  column "leaf_input" of relation "ordered_entry_index" does not exist, skipping"